### ISSUE TITLE
[CLI] Remove the automatic "Controller" suffix from the controller command

### ISSUE THREAD

**Issue by kjdelisle** [strongloop/loopback-next#886]
## Overview

As a LoopBack 4 user, I would like the CLI tool to respect my naming choice, since I might not want the word `Controller` added to the end of my class name (`Foo` -> `FooController`).

## Acceptance Criteria
- [ ] Modify all of the existing templates and logic for controller creation so that they do not append the word `Controller` to the end of the generated class name.
- [ ] Modify all existing tests to match
- [ ] Update the existing documentation to match

**Comment by virkt25** [comment]
I don't agree. I though the CLI is supposed to follow the style we recommend and I think it's a good idea to add `Controller` suffix for class names so it's clear looking at a artifact the type of the artifact.

**Comment by dhmlau** [comment]
I don't have strong opinion on either approach.  However, when I first use the cli command, i did have the puzzle why it is appending `Controller` without telling me.  

If we are appending the suffix automatically, something to consider:
- it might be good to have some hint in the prompt
- might want to be consistent when generating other artifacts (when we support them. e.g. repository, model, etc).

**Comment by bajtos** [comment]
I am personally fine with the current solution, where `Controller` is appended to the class name. I see how this can be confusing though, I like Diana's proposal to add some sort of a hint - either in the prompt or after accepting the value from the user.

**Comment by raymondfeng** [comment]
If we decide to recommend a name following a convention, we SHOULD prompt the value so that users can override it.

**Comment by virkt25** [comment]
+1 for allowing users to override the convention if they want but we keep it.

**Comment by dhmlau** [comment]
Currently if we run `lb4 controller`, we have:
```
$ lb4 controller
? Controller class name: Test
? What kind of controller would you like to generate? (Use arrow keys)
❯ Empty Controller
  REST Controller with CRUD functions
```

Are we going with:
```
$ lb4 controller
? Controller name: Test
? Controller class name: (TestController)
? What kind of controller would you like to generate? (Use arrow keys)
```
I'm not sure how we can better prompt the user for the "first name" to derive the controller class name.  Please comment.

**Comment by bajtos** [comment]
+1 for @virkt25 proposal above. I would even remove the first "Controller" word from the message, it feels redundant to me. 

_**ShoppingCartController** was created in **src/controllers/shopping-cart.controller.ts**_
